.github/workflows/deploy.yml

```diff
@@ -58,43 +58,31 @@ jobs:
 
   linux-binaries:
     strategy:
+      fail-fast: false
       matrix:
         target:
+         - i686-unknown-linux-musl
```

Comment:
this is maybe spurious, I guess? I guess also ppc could target musl for uniformity

Reply:
I'm not sure I understand your meaning. This is the build for i686 architectures; it's still commonplace to release 32-bit python wheels, which is my specific use case as the maintainer of [pillow-avif-plugin](https://github.com/fdintino/pillow-avif-plugin).

I had tried to target powerpc64le-unknown-linux-musl, but it's not trivial because that target is a [tier 3 platform](https://doc.rust-lang.org/rustc/platform-support.html#tier-3), which means that there aren't any official builds for the toolchains. So we would have to do that as part of this workflow.

Reply:
It is possible to build ppc musl if it is run in a rust-musl-cross container. See https://github.com/fdintino/cargo-c/commit/b50119686e058eea775c9f8f0656a6b892ecbfed

Reply:
Too much complexity for little gain, but thank you a lot for figuring out how to do it. :)